src/components/button/Button.tsx

```diff
@@ -112,207 +117,239 @@ const AminoButton = styled.button<ButtonProps<GroupTag>>`
+        }
+        if (p.outline) {
+          return theme.blue100;
+        }
+        return theme.buttonPrimaryHover;
+      }};
     }
-    &:active,
-    &:focus {
-      background: ${theme.blue700};
-      color: ${theme.gray0};
+    &:active {
+      background: ${p => (p.outline ? theme.blue200 : theme.blue700)};
     }
   }
 
+  &[disabled] {
+    background: ${p => (p.outline ? theme.gray0 : theme.blue400)};
+  }
+
   ${StyledSpinnerWrapper} {
-    background: ${theme.primary};
+    background: ${p => (p.outline ? theme.gray0 : theme.primary)};
   }
 `;
 
-const Secondary = styled(AminoButton)`
-  color: ${p => getAminoColor(p.color) || theme.textColor};
-  background: ${p => getAminoColor(p.background) || theme.gray100};
+const Success = styled(AminoButton)`
+  background: ${p =>
+    getAminoColor(p.background) || p.outline ? theme.gray0 : theme.success};
+  color: ${p =>
+    getAminoColor(p.color) || p.outline ? theme.success : theme.gray0};
+  box-shadow: ${p => (p.outline ? 'none' : theme.shadowButtonSuccess)};
+
+  ${p =>
+    p.outline
+      ? `border: 1px solid ${theme.green300}; border-radius: ${theme.radius6};`
+      : ''}
 
   &:not([disabled]) {
     &:hover {
-      background: ${p => getAminoColor(p.hoverBackground) || theme.gray200};
-    }
-    &:active {
-      background: ${theme.blue100};
-      color: ${theme.blue600};
-      svg path {
-        fill: currentColor;
+      background: ${p => {
+        if (p.hoverBackground) {
+          return getAminoColor(p.hoverBackground);
+        }
+        if (p.outline) {
+          return theme.green100;
+        }
+        return theme.buttonSuccessHover;
+      }};
+
+      &:active {
+        background: ${p => (p.outline ? theme.green200 : theme.green700)};
       }
     }
   }
-
-  ${StyledSpinnerWrapper} {
-    background: ${theme.gray100};
+  &[disabled] {
+    background: ${p => (p.outline ? theme.gray0 : theme.green400)};
   }
 
-  /** Night mode */
-  &.night {
-    color: ${theme.gray0};
-    background: ${theme.gray1000};
-
-    &:not([disabled]) {
-      &:hover {
-        background: ${p => getAminoColor(p.hoverBackground) || theme.gray900};
-      }
-      &:active,
-      &:focus {
-        background: ${theme.blue1000};
-        color: ${theme.blue300};
-        svg path {
-          fill: currentColor;
-        }
-      }
-    }
-
-    ${StyledSpinnerWrapper} {
-      background: ${theme.gray1000};
-    }
+  ${StyledSpinnerWrapper} {
+    background: ${p => (p.outline ? theme.gray0 : theme.success)};
   }
 `;
 
-const Danger = styled(AminoButton)`
-  background: ${p => getAminoColor(p.background) || theme.red600};
-  color: ${p => getAminoColor(p.color) || theme.gray0};
+const Standard = styled(AminoButton)`
+  color: ${p => getAminoColor(p.color) || theme.textColor};
+  background: ${p => getAminoColor(p.background) || theme.surfaceColor};
+  border: 1px solid ${p => (p.outline ? theme.gray300 : `${theme.gray600}1F`)};
```

Comment:
I don't think adding the letter after the css variable will work 🤔 Does this work?

Reply:
It does! I did a bunch of playing around with it. It was a really interesting discovery.

Reply:
It does! But only in certain circumstances. The color variables, for example, are just the hex code string. This one, `gray600` is the same as writing `'#6A6D77'` and because I'm dropping that in a string, following it up with `1F` creates `'#6A6D771F'` which add the transparency to the border. I was pretty happy to discover that. We can see a similar effect in the `Spinner.tsx` component with the border sizes: following up a string interpolation with more text (in that case, the `px` after a calculation based on the props).